pkg/sqlqueries/system_intake_grb_discussions.go

```diff
@@ -0,0 +1,25 @@
+package sqlqueries
+
+import _ "embed"
+
+//go:embed SQL/system_intake_grb_discussions/insert_internal.sql
```

Comment:
curious why `internal` suffix on file names - non blocking

Reply:
My guess is this might just be a reflection of some of the naming used on the frontend?

![image](https://github.com/user-attachments/assets/d9d6a61e-ac06-4281-ac0f-cb9a07407e8a)

I'd be down to rename to remove the `_internal` if we want, since there's currently no "External" (or any other) discussion boards.

If we _do_ make another one, we'd probably put the "Internal" queries in one folder, and the "External" queries in another.

@samoddball You wanna make a PR to just do those name changes?

Reply:
not a huge deal! i was curious more if the sql statements were for internal use only for a certain part of the codebase or service, but this makes sense! thanks!